asteroids_code/BoilerPlate/mathUtilities.hpp

```diff
@@ -1,6 +1,6 @@
 #pragma once
 
-const float pi = 3.14159265359;
+const float pi = 3.14159265359f;
```

Comment:
Please read the style guides I provided, in all of them, the constants are declared in CAPS.